src/components/hero/HeroSection.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
Small change: on the mobile version, could you make the "Register" button left aligned and fully rounded like the Figma?
<img width="294" alt="Screenshot 2025-01-20 at 12 21 29 AM" src="https://github.com/user-attachments/assets/c0d17b89-3885-4580-999b-5e5b36f20f01" />